src/cmd/list.go

```diff
@@ -121,35 +129,22 @@ func list(cmd *cobra.Command, args []string) error {
 }
 
 func getContainers() ([]toolboxContainer, error) {
-	logrus.Debug("Fetching containers with label=com.github.containers.toolbox=true")
-	args := []string{"--all", "--filter", "label=com.github.containers.toolbox=true"}
-	containers_old, err := podman.GetContainers(args...)
-	if err != nil {
-		return nil, fmt.Errorf("failed to list containers with label=com.github.containers.toolbox=true: %w", err)
-	}
+	var err error
+	var args []string
+	var containers []map[string]interface{}
+	var toolboxContainers []toolboxContainer
 
-	logrus.Debug("Fetching containers with label=com.github.debarshiray.toolbox=true")
-	args = []string{"--all", "--filter", "label=com.github.debarshiray.toolbox=true"}
-	containers_new, err := podman.GetContainers(args...)
+	logrus.Debug("Fetching all containers")
+	args = []string{"--all", "--sort", "names"}
+	containers, err = podman.GetContainers(args...)
 	if err != nil {
-		return nil, fmt.Errorf("failed to list containers with label=com.github.debarshiray.toolbox=true: %w", err)
+		return nil, fmt.Errorf("failed to get containers: %w", err)
```

Comment:
Bubbling the errors upwards is fine, but we should be careful not show some weird gibberish to the user under normal operation. In such cases, it's better to either dump the error in the debug log; or detect specific errors, at least the most common ones, and have nicer equivalents for those.

Reply:
You've got a point here. Should I address that immediately? I think there are several places like this across the code base.

Reply:
Never mind, I think I meant this comment for some other change. I see that we were bubbling the errors up even before, and I just did a quick test and it all looks good.

Sorry about the noise.